### PR TITLE
updated the spec files to include the changes requested by Rob G

### PR DESCRIPTION
**Pull request by tjhmoore**

### CODE REVIEW

File: specification/electronic-prescription-service-api.yaml [hunk 1]
```diff
@@ -18,12 +18,11 @@ info:
...
     * encode prescription data
-    * create a secondary care (outpatient) prescription
-    * cancel a secondary care (outpatient) prescription
+    * create and cancel a secondary care (outpatient) prescription
+    * create and cancel a primary care prescription
 
     You cannot currently use this API to:
-    * create a primary care prescription
-    * create a secondary care (other than outpatient) prescription
+    * create a or cancel secondary care (homecare) prescription
```
Comment (booshibooshi) [inline specification/electronic-prescription-service-api.yaml:25]: is this supposed to be `and/or` and if so, would it be the same above under `You can:` ?

File: specification/electronic-prescription-service-api.yaml [hunk 2]
```diff
@@ -18,12 +18,11 @@ info:
     ### As a prescriber
     You can:
     * encode prescription data
-    * create a secondary care (outpatient) prescription
-    * cancel a secondary care (outpatient) prescription
+    * create and cancel a secondary care (outpatient) prescription
+    * create and cancel a primary care prescription
```
Comment (callumtipper-IW) [inline specification/electronic-prescription-service-api.yaml:23]: Would these be better as individual bullets? e.g.
 * create a secondary care (outpatient) prescription
 * cancel a secondary care (outpatient) prescription
 * create a primary care prescription
 * cancel a primary care prescription

File: specification/electronic-prescription-service-api.yaml (hunk 1 above)
Comment (callumtipper-IW) [inline specification/electronic-prescription-service-api.yaml:25]: Same as above, separate bullets? 
Also, an extra "a" in "create or cancel"

Comment (tjhmoore) [reply specification/electronic-prescription-service-api.yaml:25]: yeh it was, I will update

Comment (tjhmoore) [reply specification/electronic-prescription-service-api.yaml:25]: I combined them in maybe a failed attempt to shorted the document slightly as this top section is so long! Can split them again if we think its better?

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -54,7 +50,9 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
+    The immediate EPS development priority is to support the role out of the next gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR compliant prescription tracker into the next gen APIs.
```
Comment (mfjarvis) [inline specification/electronic-prescription-service-api.yaml:53]: Should be `rollout` (or `roll-out`), not `role out`

Comment (mfjarvis) [reply specification/electronic-prescription-service-api.yaml:53]: I think `next-gen` and `FHIR-compliant` should be hyphenated

File: specification/electronic-prescription-service-api.yaml [hunk 4]
```diff
@@ -54,7 +55,9 @@ info:
     - [Personal Demographics Service - FHIR API](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) - use this to obtain patient data
 
     ## API status and roadmap
-    To see our roadmap, or to suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
+    The immediate EPS development priority is to support the rollout of the next-gen FHIR APIs for primary care and secondary care prescribing and dispensing. Alongside this we will be working on integrating a new FHIR-compliant prescription tracker into the next gen APIs.
+    
+    To see suggest, comment or vote on features for this API, see our [interactive product backlog](https://nhs-digital-api-management.featureupvote.com/?tag=eps-fhir-api).
```
Comment (callumtipper-IW) [inline specification/electronic-prescription-service-api.yaml:60]: "To see suggest, comment or vote on features for this API" should this be "To see our roadmap,..." as in the previous version ?

Comment (tjhmoore) [reply specification/electronic-prescription-service-api.yaml]: Ive just removed 'see' from this as the interactive product backlog doesn't really have much to do with out roadmap at the moment tbh

**Review by callumtipper-IW — APPROVED** [review]

**Pull request merged**